src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
Removing `.originalEvent` should do the trick I think, or the previous proposal of using a new `jQuery Event` as well. It would depend on whether the event handler functions that are attached by jQuery Mobile or its clients expect an `event.originalEvent` to be there when they are called.

Note that the whole special event system has very few user-serviceable parts and `jQuery.event.simulate` isn't documented. It was primarily built as a way to normalize browser behaviors inside jQuery core, not as a public extension mechanism.